.blah/README.ejs

```diff
@@ -0,0 +1,116 @@
+ * comparing the performance of a set of functions for a specific input (e.g. different implementations of same functionality)
+ * comparing the performance of a set of functions for the same set of inputs
+
+Benchtable is not a new JavaScript benchmarking framework -- it extends the already excellent [benchmark.js](https://github.com/bestiejs/benchmark.js) framework.
+Benchmarks are executed using benchmark.js and displayed using [cli-table](https://github.com/LearnBoost/cli-table).
+
+![Screenshot](/screenshot.png)
+
+## Features
+
+ * a Benchtable suite is an extension (subclass) of the [Benchmark.js Suite API](http://benchmarkjs.com/docs#Suite)
+ * a simple API for defining benchmarks functions and inputs
+ * display of benchmarking results as an ASCII table, with ops/sec measure used for table data
+ * simple visual identification of functions with best and worst performance for each input using color highlighting (red = worst, green = best)
+
+## Installation
+
+```sh
+npm i --save <%- _.pack.name %>
+```
+
+## API and usage
+`BenchTable` is a class extended from the benchmark.js's [`Benchmark.Suite`](http://benchmarkjs.com/docs#Suite) class - all
+functions exposed by `Benchmark.Suite` are also provided by Benchtable and therefore a Benchtable object may be used like
+an ordinary `Benchmark.Suite`.
+
+The `Benchmark.Suite.prototype.add` method can be used to add functions to be benchmarked, and while such functions will
+indeed be benchmarked with Benchtable–their results will not be shown in the results ASCII table.
+
+Please use the `BenchTable`'s `addFunction` and `addInput` methods for specifying BenchTable benchmarks, as shown below.
+
+<% docs = docs.split("\n").slice(3).join("\n"); %>
+<%- docs %>
+
+### Events
+
+#### `complete`
+Get results table.
+
+After all benchmarks are finished, the `complete` event is fired and the `cli-table` table instance with the results is
+available through the `BenchTable.prototype.table` property.
+
+The ASCII string serialization of the table is obtained by calling `.toString()` on that object.
+
+```js
+var suite = new BenchTable();
+...
+suite.on('complete', () => {
+  suite.table.toString();
+});
+```
+
+## Examples
+
+This is the Benchtable version of the example from the [Benchmark.js homepage](http://benchmarkjs.com).
+Also available in the [examples directory](/examples/string_search.js).
+
+### Default behavior
+
+```js
+<%- getExample("string_search.js") %>
+```
+
+## Transposed table
+
+If the Benchtable object is initialized with the `{isTransposed : true}` option (see [this example](/examples/string_search_transposed.js)), the script produces the same output but with rows and columns transposed:
+
+```js
+<%- getExample("string_search_transposed.js") %>
+```
+
+## Contributing
+
+Want to contribute to this project? See information [here](CONTRIBUTING.md).
+
+## Credits
+Benchtable is developed by [Ivan Zuzak](http://ivanzuzak.info) \<izuzak@gmail.com\>;.
```

Comment:
There is already an issue regarding that: https://github.com/atom/markdown-preview/issues/320